tangelo/linq/target/backend.py

```diff
@@ -224,24 +234,29 @@ def simulate(self, source_circuit, return_statevector=False, initial_statevector
                 statevector[0] = 1.0
             return (frequencies, statevector) if return_statevector else (frequencies, None)
 
+        # For mid-circuit measurements post-process the result
         if save_mid_circuit_meas:
             # TODO: refactor to break a circular import. May involve by relocating get_xxx_oneterm functions
```

Comment:
Is the circular import still there?

Reply:
I think so. Right now it's worked around by importing the offending module locally. Fixing it is not really directly related to this PR but rather design choices in tangelo.